src/components/Categories/Card/Card.css.js

```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components';
+
+export const CategoryCardImage = styled.img`
+  width: 100%;
+  margin-bottom: 16px;
+`;
+
+export const CategoryName = styled.h6`
+  margin-left: 8px;
+  margin-right: 8px;
+  margin-bottom: 8px;
```

Comment:
Creo que esta línea la podríamos cambiar por la sintaxis de margin completo donde ponemos 4 valores de cada uno de los lados, en orden arriba, derecha, abajo, izquierda, así como en sentido de manecillas del reloj.

```suggestion
  margin: 0 8px 8px 8px
```

Básicamente le estamos diciendo que queremos 0px de margin-top, 8px de right, 8px de bottom, 8px de left.